sentry-core/src/main/java/io/sentry/core/EnvelopeSender.java

```diff
@@ -214,8 +214,6 @@ private void processEnvelope(final @NotNull SentryEnvelope envelope, final @Null
               items);
           break;
         }
-      } else {
```

Comment:
Is this needed to changing to sentry-core? Or just another refactor?

Reply:
this class needed to be changed, so it made sense to "clean up" dead code, it was a useless logging